jinahub/indexers/indexer/LMDBIndexer/__init__.py

```diff
@@ -55,10 +54,11 @@ def index(self, docs: DocumentArray, parameters: Dict, **kwargs):
         :param docs: the documents to add
         :param parameters: parameters to the request
         """
-        trav_path = parameters.get('traversal_path', self.default_traversal_path)
+        trav_paths = parameters.get('traversal_paths', self.default_traversal_paths)
```

Comment:
I'm not a big fan of abbreviations. Can we keep the full name?

Reply:
sure, done